Include/internal/pycore_interp.h

```diff
@@ -176,6 +177,7 @@ struct _is {
     struct _Py_async_gen_state async_gen;
     struct _Py_context_state context;
     struct _Py_exc_state exc_state;
+    struct _Py_immutability_state immutability;
```

Comment:
Does this mean that different interpreters have different views on what is freezable?